modules/BlueskyClip/ViewController.swift

```diff
@@ -3,7 +3,7 @@ import WebKit
 import StoreKit
 
 class ViewController: UIViewController, WKScriptMessageHandler, WKNavigationDelegate {
-  let defaults = UserDefaults(suiteName: "group.app.bsky")
+  let defaults = UserDefaults(suiteName: "group.app.deersocial")
```

Comment:
can we make all of these `group.social.deer` or `group.social.deer.app` so it aligns with the domain we actually have?